libraries/AC_Fence/AC_Fence.h

```diff
@@ -251,6 +251,7 @@ class AC_Fence
     AP_Int8         _ret_rally;             // return to fence return point or rally point/home
     AP_Int16        _ret_altitude;          // return to this altitude
     AP_Int16        _options;               // options bitmask, see OPTIONS enum
+    AP_Float        _notify_freq;    // margin notification frequency
```

Comment:
personally I'd probably make it an AP_Int8 but not a blocker

Reply:
The notify interval can't be an int as it needs to go lower than 1 hz - alternatives are changing to a ms interval, but I don't know which is more acceptable (or deci-hertz - ug)